### PR TITLE
refactor: verified user badge

### PR DESCRIPTION
**Pull request by chrismclarke**
PR Checklist

- [ ] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [x] - Latest `master` branch merged

PR Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Refactor code that creates and provides info for verified user badges. Currently all logic for these badges are managed by various stores (e.g. user store, howto store, event store) that retrieve lists of verified users and pass down through components as required. This involves quite a lot of copy-paste and prop drilling so isn't ideal.

Instead this pr proposes a single component `<VerifiedUserBadge />` which retrieves info from the aggregation store to check if a provided user is verified and displays badge accordingly

## Review Notes
Would be useful to check anywhere verified badges appeared previously just to check they still display as intended

## Git Issues

Closes #

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on slack in the `platform-dev` channel.

### CODE REVIEW

File: src/components/Comment/CommentHeader.tsx [hunk 1]
```diff
@@ -30,9 +27,12 @@ export const CommentHeader = ({
           >
             {creatorName}
           </Link>
-          {verified && (
-            <Image src={VerifiedBadgeIcon} ml={1} height="12px" width="12px" />
-          )}
+          <VerifiedUserBadge
+            userId={creatorName}
```
Comment (chrismclarke) [inline src/components/Comment/CommentHeader.tsx:31]: Most component cases have dropped passing verified prop and manual badge render in favour of common component

File: src/pages/Howto/Content/Howto/Howto.tsx [hunk 2]
```diff
@@ -125,18 +127,17 @@ export class Howto extends React.Component<
...
         <>
           <HowtoDescription
             howto={activeHowto}
-            verified={
-              this.injected.userStore.verifiedUsers[activeHowto._createdBy]
-            }
-            votedUsefulCount={
-              this.injected.userStore.userVotedHowtos[activeHowto._id]
-            }
+            votedUsefulCount={users_votedUsefulHowtos[activeHowto._id]}
```
Comment (chrismclarke) [inline src/pages/Howto/Content/Howto/Howto.tsx:140]: the votedUsefulCount aggregation hasn't been refactored (just verified user badge for now) to keep scope slightly smaller (but may be useful to update in a similar way to handle counter at a component level)

**Review by davehakkens — APPROVED** [review]
looking good here!

**Pull request merged**